resources/actions/nexus.py

```diff
@@ -0,0 +1,330 @@
+    else:
+        logging.error('Failed to get repository: %s' % name)
+        logging.error('Status code: %s' % response.status_code)
+        logging.error('Response: %s' % response.content)
+        sys.exit(1)
+
+
+def create_apt_repo(url, name, user, password,
+                    blob_store, write_policy,
+                    distribution, keypair=None, keypair_file=None,
+                    passphrase=None):
+
+    api_url = "%s/service/rest/v1/repositories/apt/hosted" % url
+
+    if keypair_file:
+        keypair = keypair_file.read()
+
+    params = {
+          "name": name,
+          "online": "true",
+          "storage": {
+            "blobStoreName": blob_store,
+            "strictContentTypeValidation": "true",
+            "writePolicy": write_policy
+          },
+          "cleanup": {
+            "policyNames": [
+              "string"
+            ]
+          },
+          "component": {
+            "proprietaryComponents": "true"
+          },
+          "apt": {
+            "distribution": distribution
+          },
+          "aptSigning": {
+            "keypair": keypair,
+            "passphrase": passphrase
+          }
+    }
+
+    logging.info('Creating hosted APT repository: %s' % name)
+    response = requests.post(api_url, json=params,
+                             auth=requests.auth.HTTPBasicAuth(user, password))
+
+    if response.status_code == 201:
+        logging.info('Done')
+    else:
+        logging.error('Failed to create repository: %s' % name)
+        logging.error('Status code: %s' % response.status_code)
+        logging.error('Response: %s' % response.content)
+        sys.exit(1)
+
+def upload_yum(url, name, file_path, user, password, upload_path=None):
+    try:
+        artifcat = open(file_path, 'rb').read()
+    except IOError:
+        logging.error('Unable to open file: %s' % file_path)
+        sys.exit(1)
+
+    fn = file_path.split('/')[-1]
+
+    upload_url = '/'.join(item.strip('/') for item in [url, 'repository', name])
+
+    if upload_path:
+        upload_url = '/'.join(item.strip('/') for item in [upload_url, upload_path])
+
+    upload_url = '/'.join([upload_url, fn])
+
+    response = requests.put(upload_url, data=artifcat,
+                            auth=requests.auth.HTTPBasicAuth(args.user, args.password))
+    if response.status_code != 200:
+        logging.error('Unable to upload artifact %s to %s' % (file_path, upload_url))
+        logging.error(response.content)
+        logging.error(response.status_code)
+        sys.exit(1)
+
+
+def upload_apt(url, name, file_path, user, password):
+    try:
+        artifcat = open(file_path, 'rb')
+    except IOError:
+        logging.error('Unable to open file: %s' % file_path)
+        sys.exit(1)
+
+    upload_url = '/'.join(item.strip('/') for item in [url, 'repository', name])
+
+    if not upload_url.endswith('/'):
+        upload_url = upload_url + '/'
+
+    response = requests.post(upload_url, data=artifcat,
```

Comment:
PEP-8 (and flake8 tool) recommends the following formatting (for example):
```python
    response = requests.post(
        upload_url, data=artifcat,
        auth=requests.auth.HTTPBasicAuth(args.user, args.password)
    )
```
How to use (just in case):
```sh
pip3 install flake8
flake8 nexus.py
```

Reply:
Great, thank you
Fixed all warnings using flake8